geopandas/tests/test_sindex.py

```diff
@@ -670,6 +684,47 @@ def test_query_bulk_sorting(self, sort, expected):
                 )
             raise e
 
+    # ------------------------- `nearest` tests ------------------------- #
+    @pytest.mark.skipif(
+        TEST_NEAREST,
+        reason=(
+            f"PyGEOS >= {MIN_PYGEOS_VERSION_FOR_NEAREST}"
+            " must be installed and activated via the geopandas.options module to"
+            " test sjoin_nearest"
+        ),
+    )
+    def test_no_nearest(self):
+        df = geopandas.GeoDataFrame({"geometry": []})
+        with pytest.raises(
+            AttributeError, match="Expected the spatial index to implement `nearest`"
+        ):
+            df.sindex.nearest(Point(0, 0))
+
+    @pytest.mark.skipif(
+        not TEST_NEAREST,
+        reason=(
+            f"PyGEOS >= {MIN_PYGEOS_VERSION_FOR_NEAREST}"
```

Comment:
```suggestion
            f"Pygeos >= 0.10"
```

Reply:
As above, I've simpified the check and `reason` string.